bitnami/etcd/3.5/debian-11/rootfs/opt/bitnami/scripts/libetcd.sh

```diff
@@ -664,6 +664,13 @@ etcd_initialize() {
             if is_boolean_yes "$ETCD_DISABLE_PRESTOP"; then
                 info "The member will try to join the cluster by it's own"
                 export ETCD_INITIAL_CLUSTER_STATE=existing
+                #
```

Comment:
Could you change the comment:

```
# If ETCD_DISABLE_PRESTOP is set, we won't dynamically adjust membership. In this case
# we'll return and allow etcd to start and join the statically configured cluster.
```

Reply:
you bet -- changing..

Reply:
changed!

Reply:
@mdhont any additional changes we need here?